sal_interfaces/WeatherStation/WeatherStation_Telemetry.xml

```diff
@@ -339,21 +339,21 @@
       <EFDB_Name>paAvg1M</EFDB_Name>
       <Description>1 minute average value</Description>
       <IDL_Type>double</IDL_Type>
-      <Units>pa</Units>
+      <Units>hPa</Units>
```

Comment:
See for pressure value..this is hPa (preferred?), above is pa. pa isn't astropy unit, correct is Pa.